### PR TITLE
raidboss: improve lakshmi extreme timeline

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/04-sb/trial/lakshmi-ex.txt [hunk 1]
```diff
@@ -11,96 +11,99 @@ hideall "Alluring Arm"
...
-161.5 "The Pull Of Light" # end of castbar
-171.7 "Stotram" # end of castbar
-179.0 "The Pall Of Light" # end of castbar
-188.2 "The Pull Of Light" # end of castbar
+107.2 "Hand Of Grace" sync /:Lakshmi:214C:/ window 110,0
+114.4 "The Pull Of Light" sync /:Lakshmi:215E:/
+121.4 "Blissful Arrow" sync /:Lakshmi:214F:/
+122.0 "Blissful Spear" sync /:Lakshmi:2151:/
+124.6 "Stotram" sync /:Lakshmi:2147:/
+135.8 "The Pull Of Light" sync /:Lakshmi:215E:/
```
Comment (JLGarber) [inline ui/raidboss/data/04-sb/trial/lakshmi-ex.txt:31]: Any chance of adding clarifications on Pull/Pall/Path as translations, to match Normal mode?

File: ui/raidboss/data/04-sb/trial/lakshmi-ex.txt [hunk 2]
```diff
@@ -11,96 +11,99 @@ hideall "Alluring Arm"
...
-485.2 "The Pull Of Light" # end of castbar
-488.5 "--chanchala end--"
+441.7 "--sync--" sync /:2148:Lakshmi/ window 50,10
+444.7 "Chanchala #2" sync /:Lakshmi:2148:/
+458.5 "Divine Doubt" sync /:Lakshmi:214A:/
+473.9 "The Path Of Light" sync /:Lakshmi:215B:/
+484.0 "The Pull Of Light" sync /:Lakshmi:215F:/
+486.0 "--chanchala end--"
+509.5 "The Pull Of Light" sync /:Lakshmi:215E:/
+511.5 "Blissful Spear (Z)" sync /:Lakshmi:2166:/
```
Comment (JLGarber) [inline ui/raidboss/data/04-sb/trial/lakshmi-ex.txt:75]: Here and elsewhere, what do the parentheses mean?

Comment (quisquous) [reply ui/raidboss/data/04-sb/trial/lakshmi-ex.txt:75]: These are the patterns that the vril show up in.  In extreme, there's ground aoes (lines in a Z pattern, circle on the outside, etc) with vril in them.

File: ui/raidboss/data/04-sb/trial/lakshmi-ex.txt (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/trial/lakshmi-ex.txt:31]: Done!

File: ui/raidboss/data/04-sb/trial/lakshmi-ex.txt (hunk 2 above)
Comment (JLGarber) [reply ui/raidboss/data/04-sb/trial/lakshmi-ex.txt:75]: Huh, okay. I didn't really pay attention to the exact patterns going through, that's useful stuff.

**Pull request merged**